samples/frameworks/react/ts/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "typescript-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/event-hubs": "^5.3.1",
+    "@azure/identity": "^1.2.0",
+    "@azure/service-bus": "^7.0.0",
+    "@azure/storage-blob": "^12.3.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
+    "typescript": "^4.0.3",
+    "uuid": "^8.3.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
```

Comment:
What's going on here?
React needs this?

Reply:
The browserslist gets added by create-react-app. It's a generic way to specify what browsers this app supports without listing every single one. For example, if you were to run `npx browserslist` from the same directory this is listed you'll see a list of browsers that this app is targeting / supporting. 

I unfortunately know very little more than that 😄 but I think it's used by various polyfills to figure out what they need to prefix.